node/src/chain_spec.rs

```diff
@@ -124,16 +136,219 @@ pub fn local_testnet_config() -> Result<ChainSpec, String> {
+				)
+				.into(),
+				public_from_ss58::<ed25519::Public>(
+					"2PLiyfMnuEc7mcgSqfqA7ZHstYeQh3kVZ8eJrsUcYsqTKU3W",
+				)
+				.into(),
+			),
+			(
+				public_from_ss58::<sr25519::Public>(
+					"2Jhqi21p3UdGu1SBJzeUyQM9FudC5iC7e4KryAuJ4NZMhYPe",
+				)
+				.into(),
+				public_from_ss58::<ed25519::Public>(
+					"2LCKNXvVSWpL6rBusK2RUkYaFV1wv9MnWG2UpGQecsrKpp4R",
+				)
+				.into(),
+			),
+			(
+				public_from_ss58::<sr25519::Public>(
+					"2JwCMVvx8DgzpRD7sF1PKpzCDbmGiB2oa67ka2SuUe2TSJgB",
+				)
+				.into(),
+				public_from_ss58::<ed25519::Public>(
+					"2P4Bbk7edF41ny7FSMrQ6u2UsjoTaDhD1DARzwdkeDwBiZfn",
+				)
+				.into(),
+			),
+		]
+	}
+}
+
+struct CrannyKeys;
+
+impl CrannyKeys {
+	fn root() -> AccountId {
+		public_from_ss58::<sr25519::Public>("5CVcJfKKo7uqMGvAE9fzqw66tEfngwJat5FruAsa6hbSkejD")
+			.into()
+	}
+	fn authorities() -> Vec<(AuraId, GrandpaId)> {
+		vec![
+			(
+				public_from_ss58::<sr25519::Public>(
+					"5DDBqKzDw4GnEVmqRXvo8iiWzFxT76E3KUDTk79NnM9F6B8V",
+				)
+				.into(),
+				public_from_ss58::<ed25519::Public>(
+					"5CyuN5TUy6hd1WN2o3uZLpRoBjsAqzXLxUFD2GNT1Sjv3sS5",
+				)
+				.into(),
+			),
+			(
+				public_from_ss58::<sr25519::Public>(
+					"5GhK3Hm39J7yL6ZYoeUxynhfTkPxCd3EqnAPfgHcDo37wqmz",
+				)
+				.into(),
+				public_from_ss58::<ed25519::Public>(
+					"5FBqLTmuJWUFkceoeyWRqrSkYpuqJi9hXNAFFfLL3oTJzSCp",
+				)
+				.into(),
+			),
+			(
+				public_from_ss58::<sr25519::Public>(
+					"5DHwmxfN57NvGpLYFFfxrshnGxccE12VbUGsFCzGSYZQKMfD",
+				)
+				.into(),
+				public_from_ss58::<ed25519::Public>(
+					"5DpXQisSziSLWvRKBPH4F8Twdg89gnKbYpMQDtGmgTJrEzyr",
+				)
+				.into(),
+			),
+		]
+	}
+}
+
+pub fn integritee_mainnet_fresh_config() -> Result<ChainSpec, String> {
+	integritee_chain_spec(
+		"Integritee Mainnet",
+		"integritee-mainnet",
+		GenesisKeys::Integritee,
+		r#"{
+		"ss58Format": 13,
+		"tokenDecimals": 12,
+		"tokenSymbol": "TEER"
+		}"#,
+	)
+}
+
+pub fn cranny_fresh_config() -> Result<ChainSpec, String> {
+	integritee_chain_spec(
+		"Cranny",
+		"cranny",
+		GenesisKeys::Cranny,
+		r#"{
+		"ss58Format": 42,
+		"tokenDecimals": 12,
+		"tokenSymbol": "CRA"
+		}"#,
+	)
+}
+
+pub fn integritee_mainnet_config() -> Result<ChainSpec, String> {
+	ChainSpec::from_json_bytes(&include_bytes!("../res/integritee-mainnet.json")[..])
+}
+
+pub fn cranny_config() -> Result<ChainSpec, String> {
+	ChainSpec::from_json_bytes(&include_bytes!("../res/cranny.json")[..])
+}
+
+pub fn integritee_chain_spec(
+	chain_name: &str,
+	chain_id: &str,
+	genesis_keys: GenesisKeys,
+	token_specs: &str,
+) -> Result<ChainSpec, String> {
+	let (root, endowed, authorities) = match genesis_keys {
+		GenesisKeys::Integritee =>
+			(IntegriteeKeys::root(), vec![IntegriteeKeys::root()], IntegriteeKeys::authorities()),
+		GenesisKeys::Cranny =>
+			(CrannyKeys::root(), vec![CrannyKeys::root()], CrannyKeys::authorities()),
+	};
+
+	// Todo: Chris check wasm binary
```

Comment:
is this question answered?

Reply:
Not yet, needs @clangenb to answer.

Reply:
I wondered why they call it 'dev'-`WASM_BINARY`. I wanted to make sure that we don't include something that is not meant for production. But it seems fine. So we can remove the comment.